modin/pandas/dataframe.py

```diff
@@ -11,6 +11,17 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+Implement DataFrame public API as Pandas does.
+
+Almost all docstrings for public and magic methods should be inherited from Pandas
+for better maintability. So some codes are ignored in pydocstyle check:
+    - D101: missing docstring in class
+    - D102: missing docstring in public method
+    - D105: missing docstring in magic method
+Manually add documentation for private methods only.
```

Comment:
If we add documentation for private methods, it will affect us ([see your message](https://github.com/modin-project/modin/pull/2196#discussion_r502357977)). Isn't 't? As I think, we should only add docstrings for those methods which are not presented in pandas.